http/README.md

```diff
@@ -13,10 +13,10 @@ The service is configured using the environment variables presented in the follo
 | MG_HTTP_ADAPTER_PORT             | Service HTTP port                                                                  | 80                                  |
 | MG_HTTP_ADAPTER_SERVER_CERT      | Path to the PEM encoded server certificate file                                    | ""                                  |
 | MG_HTTP_ADAPTER_SERVER_KEY       | Path to the PEM encoded server key file                                            | ""                                  |
-| MG_THINGS_AUTH_GRPC_URL          | Things service Auth gRPC URL                                                       | <localhost:7000>                    |
-| MG_THINGS_AUTH_GRPC_TIMEOUT      | Things service Auth gRPC request timeout in seconds                                | 1s                                  |
-| MG_THINGS_AUTH_GRPC_CLIENT_CERT  | Path to the PEM encoded things service Auth gRPC client certificate file           | ""                                  |
-| MG_THINGS_AUTH_GRPC_CLIENT_KEY   | Path to the PEM encoded things service Auth gRPC client key file                   | ""                                  |
+| MG_CLIENTS_AUTH_GRPC_URL          | Things service Auth gRPC URL                                                       | <localhost:7000>                    |
+| MG_CLIENTS_AUTH_GRPC_TIMEOUT      | Things service Auth gRPC request timeout in seconds                                | 1s                                  |
+| MG_CLIENTS_AUTH_GRPC_CLIENT_CERT  | Path to the PEM encoded things service Auth gRPC client certificate file           | ""                                  |
+| MG_CLIENTS_AUTH_GRPC_CLIENT_KEY   | Path to the PEM encoded things service Auth gRPC client key file                   | ""                                  |
 | MG_THINGS_AUTH_GRPC_SERVER_CERTS | Path to the PEM encoded things server Auth gRPC server trusted CA certificate file | ""                                  |
 | MG_MESSAGE_BROKER_URL            | Message broker instance URL                                                        | <nats://localhost:4222>             |
```

Comment:
```suggestion
| MG_CLIENTS_AUTH_GRPC_URL          | Clients service Auth gRPC URL                                                       | <localhost:7000>                    |
| MG_CLIENTS_AUTH_GRPC_TIMEOUT      | Clients service Auth gRPC request timeout in seconds                                | 1s                                  |
| MG_CLIENTS_AUTH_GRPC_CLIENT_CERT  | Path to the PEM encoded clients service Auth gRPC client certificate file           | ""                                  |
| MG_CLIENTS_AUTH_GRPC_CLIENT_KEY   | Path to the PEM encoded clients service Auth gRPC client key file                   | ""                                  |
| MG_CLIENTS_AUTH_GRPC_SERVER_CERTS | Path to the PEM encoded clients server Auth gRPC server trusted CA certificate file | ""                                  |
```

Then please change in line https://github.com/dborovcanin/magistrala/blob/7a2274d48234c17ba835427efaf49de4dcdddc75/http/README.md#L31-L32